compiler/rustc_expand/src/base.rs

```diff
@@ -697,6 +697,9 @@ pub struct SyntaxExtension {
     pub allow_internal_unsafe: bool,
     /// Enables the macro helper hack (`ident!(...)` -> `$crate::ident!(...)`) for this macro.
     pub local_inner_macros: bool,
+    /// Should debuginfo for the macro be collapsed to the outermost expansion site (in other
+    /// words, was the macro definition annotated with `#[collapse_debuginfo]`)?
+    pub collapse_debuginfo: bool,
```

Comment:
Should we use bitflags for `allow_internal_unsafe`, `local_inner_macros` and `collapse_debuginfo`?

Reply:
Switched to using bitflags for this - decided to share the bitflags between `SyntaxExtension` and `ExpnData`, so ended up undoing the changes to `ExpnKind` that I had.